libs/watt/src/lib/components/vater/vater-flex.component.ts

```diff
@@ -49,6 +49,18 @@ export class VaterFlexComponent {
   @HostBinding('style.flex-direction')
   direction: Direction = 'column';
 
+  @Input()
+  @HostBinding('style.--grow')
```

Comment:
Question: Why not refactor to use signals? :-)

Reply:
I will, eventually 😄 I'm trying to keep the PR's light